python/podio_class_generator.py

```diff
@@ -151,7 +151,8 @@ def _get_filenames_templates(template_base, name):
         'ConstObject': 'Const',
         'PrintInfo': 'PrintInfo',
         'Object': '',
-        'Component': ''
+        'Component': '',
+        'SIOBlock': 'SIOBlock'
```

Comment:
```suggestion
        'SIOBlock': 'SIOBlock',
```
Avoids noisy diffs next time